relay/sources/apache/rudder-vhost.conf

```diff
@@ -46,13 +46,17 @@
 # # If you change the port for one of the virtual hosts you will likely need to add
 # # Listen 8443
 #
+# # If you already have this you can remove it
+# NameVirtualHost *:443
```

Comment:
this has no effect on apache 2.4

Reply:
And I think we don't have 2.2 anywhere

Reply:
```
In 2.3.11 and later, any time an IP address and port combination is used in multiple virtual hosts, name-based virtual hosting is automatically enabled for that address.

This directive currently has no effect.
```

https://httpd.apache.org/docs/2.4/mod/core.html#namevirtualhost

Reply:
and we already successfully split the virtual host without it, so I don't think it changes something.